app/views/staff/sponsors/edit.html.haml

```diff
@@ -0,0 +1,12 @@
+.row
+  .col-md-12
+    .page-header.clearfix
+      .btn-navbar.pull-right
+        = link_to event_staff_sponsor_path(current_event, @sponsor),
+          method: :delete, data: { confirm: 'Are you sure? Confirming this action will remove the sponsor from the event website. Are you sure you want to do this? ' + 'It can not be undone.' },
```

Comment:
I am curious why this string message is concatenated?

Reply:
Haha, I honestly didn't notice that! I repurposed a Modal message from the Program Session confirm message. 
It has no reason to be. I kind of can't believe that I didn't notice it.